packages/react-components/src/components/ChatMessage/MyMessageComponents/ChatMyMessageComponent.tsx

```diff
@@ -169,28 +171,34 @@ export const ChatMyMessageComponent = (props: ChatMyMessageComponentProps): JSX.
       if (`attachments` in message && attachments) {
         message.attachments = attachments;
       }
-      props.onUpdateMessage &&
-        message.messageId &&
-        (await props.onUpdateMessage(
-          message.messageId,
-          text,
-          /* @conditional-compile-remove(file-sharing-acs) */
-          { attachments: attachments }
-        ));
+      await onUpdateMessage?.(
+        message.messageId,
+        text,
+        /* @conditional-compile-remove(file-sharing-acs) */
+        { attachments: attachments }
+      );
       setIsEditing(false);
+      onEditComplete?.();
     },
-    [message, props]
+    [message, onEditComplete, onUpdateMessage]
   );
```

Comment:
Should we also add `onEditComplete` handler to `onResendClick`?

Reply:
Also where should the focus be after onRemoveClick is triggered?

Reply:
no, it won't work for resending. During resending, we delete the message and send it again and for UI, it will mean that an old message component will be deleted from the hirarchy and a new one will be added.